server/CVCMIServer.cpp

```diff
@@ -607,11 +608,11 @@ void handleLinuxSignal(int sig)
 }
 #endif
 
-int main(int argc, char** argv)
+int main(int argc, char * * argv)

```

Comment:
@FeniksFire yeah we certainly should keep it in mind and this is why this PR is here.
It's simply to hard to keep it all in mind and check it all for just one person.

Might be you can find some other weird bits of code here.